java/code/src/com/redhat/rhn/domain/notification/types/OnboardingFailed.java

```diff
@@ -83,7 +83,7 @@ public String getSummary() {
     @Override
     public String getDetails() {
         if (details != null) {
-            return String.format("<pre>%s</pre>", details.replaceAll("\\\\n", "\n"));
+            return String.format("<pre>%s</pre>", details.replace("\\\\n", "\n"));
```

Comment:
`\\\\n` is the regexp way to write `\\n` probably. I would love to see some unit tests to make sure the change doesn't introduce a bug here